app/components/repos-list-item.js

```diff
@@ -1,23 +1,23 @@
-import $ from 'jquery';
 import Component from '@ember/component';
 import Polling from 'travis/mixins/polling';
 import colorForState from 'travis/utils/color-for-state';
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default Component.extend(Polling, {
   tagName: 'li',
   pollModels: 'repo',
   classNames: ['repo'],
 
+  scroller: service(),
+
   color: computed('repo.currentBuild.state', function () {
     return colorForState(this.get('repo.currentBuild.state'));
   }),
 
   scrollTop() {
-    if (window.scrollY > 0) {
-      return $('html, body').animate({
-        scrollTop: 0
-      }, 200);
+    if (window && window.pageYOffset) {
+      this.scroller.scrollTo(0, 200);
```

Comment:
As far as I understand, `scrollTo` [receives 3 arguments](https://github.com/peopleconnectus/ember-concurrency-scroll#scrollerscrolltostart-end-options-function) - `start`, `end` and `options`. With this call it seems that `start = 0` and `end = 200`, which means that the service will scroll from 0 to 200 - is it correct? Because the old code seems to be scrolling to 0 from whatever scroll value was. I didn't test it, so it's mostly my assumption, but I wanted to raise it to your attention.

Reply:
Hmm good point, I think I may have misinterpreted it at some point. I'll look into it

Reply:
So yes, I think you're right and this should be `this.scroller.scrollTo(window.pageYOffset, 0)`

But I also noticed, that this function isn't ever used in this component or its template. It seems like it used to be done on clicking the component, but it was disconnected some time ago. So, it seems like it's actually safe to remove altogether

Reply:
Nice! :)